pages/privacy-policy.html

```diff
@@ -42,7 +44,7 @@
 
     <main>
       <section class="privacy-policy__header">
-        <img src="/assets/images/website-images/logo-with-background.png" />
+        <img src="/assets/images/website-images/logo-with-background.png" alt="logo-image" />
```

Comment:
same as the other two comments, we'll need to add something a bit more descriptive